docs/en/concept/schema-feature.md

```diff
@@ -172,6 +172,46 @@ constraintKeys = [
 | INDEX_KEY      | key         |
 | UNIQUE_KEY     | unique key  |
 
+## Multi table schemas
+
+```
+tables_configs = [
+  {
+    schema {
+      table = "database.schema.table1"
+      schema_first = false
+      comment = "comment"
```

Comment:
Move `table`, `schema first`, `comment` outside of schema?

Reply:
it came from https://github.com/apache/seatunnel/pull/5628, this pr doesn't change it.